dependencies/CMakeLists.txt

```diff
@@ -19,7 +19,6 @@ project( dependencies LANGUAGES NONE )
 find_package(Patch REQUIRED)
 
 add_subdirectory(	libxml2 )
```

Comment:
So we're no longer building OpenSSL in the tree, even if it's not installed on the system?

Reply:
We'll need to update BUILDING.md to account for installation of OpenSSL, too...

Reply:
No, mixing and matching is really hard. Some things in CMake are easy to do in either the configure or build stages but not in both. Yes, need to update the BUILDING.md, good catch!